enginetest/queries/priv_auth_queries.go

```diff
@@ -1338,6 +1338,98 @@ var UserPrivTests = []UserPrivilegeTest{
 			},
 		},
 	},
+	{
+		Name: "information_schema.columns table 'privileges' column gets correct values",
+		SetUpScript: []string{
+			"CREATE TABLE checks (a INTEGER PRIMARY KEY, b INTEGER, c VARCHAR(20))",
+			"CREATE TABLE test (pk BIGINT PRIMARY KEY, c VARCHAR(20), p POINT default (POINT(1,1)))",
+			"CREATE USER tester@localhost;",
+		},
+		Assertions: []UserPrivilegeTestAssertion{
+			{
+				User:     "tester",
+				Host:     "localhost",
+				Query:    "SELECT count(*) FROM information_schema.columns where table_schema = 'mydb' and table_name = 'test';",
```

Comment:
(minor) might be nice to mix some different casing for `inFORmation_ScHeMa` in, just to test case insensitivity.